core/test/martian/test_helpers.cljc

```diff
@@ -5,8 +5,8 @@
 
 #?(:clj
    (defmacro json-resource [resource-name]
-     (json/parse-string (slurp (io/resource resource-name)))))
+     (list 'quote (json/parse-string (slurp (io/resource resource-name))))))
 
 #?(:clj
    (defmacro yaml-resource [resource-name]
-     (yaml/parse-string (slurp (io/resource resource-name)))))
+     (list 'quote (yaml/parse-string (slurp (io/resource resource-name))))))
```

Comment:
I needed to quote this since the EDN representation of a flatland linked list is written like `#ordered ([:a b])` etc. Not quoting that, would evaluate the data structure as a function call.

Reply:
I'm not sure I understand, these should just be normal json and yaml data using clojure's native data structures?

Reply:
This is an issue specifically with bb:

```
user=>  (defmacro yaml [s] (doto (clj-yaml.core/parse-string s) prn))
#'user/yaml
user=>  (yaml "{a: [1 2 3]}")
#ordered/map ([:a ("1 2 3")])
class java.lang.String cannot be cast to class clojure.lang.IFn (java.lang.String is in module java.base of loader 'bootstrap'; clojure.lang.IFn is in unnamed module of loader 'app')
```

In clojure you can reproduce a similar thing with:

```
user=> (defmacro my-map [x] {:a (map inc [1 2 3])})
#'user/my-map
user=> (my-map 1)
Execution error (ClassCastException) at user/eval172 (REPL:1).
class java.lang.Long cannot be cast to class clojure.lang.IFn (java.lang.Long is in module java.base of loader 'bootstrap'; clojure.lang.IFn is in unnamed module of loader 'app')
```

Adding a quote in front of it fixes it:

```
user=> (defmacro my-map [x] (list 'quote {:a (map inc [1 2 3])}))
#'user/my-map
user=> (my-map 1)
{:a (2 3 4)}
```

It can be argued that the behavior in bb is a bug since clojure didn't have this problem, but adding a `'quote` doesn't hurt either way.